src/Tizen.Network.IoTConnectivity/Tizen.Network.IoTConnectivity/RemoteResource.cs

```diff
@@ -423,8 +423,20 @@ public async Task<RemoteResponse> GetAsync(ResourceQuery query = null)
         private static void NativeGetResultCallbackHandler(IntPtr resource, int err, int requestType, IntPtr responseHandle, IntPtr userData)
         {
             IntPtr responseCompletionId = userData;
-            TaskCompletionSource<RemoteResponse> responseCompletionSource = _taskCompletionMap[responseCompletionId];
-            _taskCompletionMap.Remove(responseCompletionId);
+            TaskCompletionSource<RemoteResponse> responseCompletionSource;
+
+            Log.Info(IoTConnectivityErrorFactory.LogTag, "Result callback for : " + responseCompletionId);
+
+            try
+            {
+                responseCompletionSource = _taskCompletionMap[responseCompletionId];
```

Comment:
You can use `TryGetValue` without exception try/catch
https://docs.microsoft.com/ko-kr/dotnet/api/system.collections.concurrent.concurrentdictionary-2.trygetvalue?view=netframework-4.8#System_Collections_Concurrent_ConcurrentDictionary_2_TryGetValue__0__1__

Reply:
Updated the code to use TryGetValue